Remove spurious slashes from manpage. Thanks to Dann Frazier for pointing it out.
Closes #572392

Author: Chris Taylor <ctaylor@debian.org>
--- a/doc/socat.1
+++ b/doc/socat.1
@@ -3667,8 +3667,8 @@ Send bug reports to <socat@dest-unreach\
 .SH "SEE ALSO"
 
 .PP 
-nc\e(1), netcat6\e(1), sock\e(1), rinetd\e(8), cage\e(1), socks\&.conf\e(5), openssl\e(1),
-stunnel\e(8), pty\e(1), rlwrap\e(1), setsid\e(1)
+nc(1), netcat6(1), sock(1), rinetd(8), cage(1), socks.conf(5), openssl(1),
+stunnel(8), pty(1), rlwrap(1), setsid(1)
 .PP 
 \fBSocat\fP home page http://www\&.dest-unreach\&.org/socat/
 .PP 
